pino.js

```diff
@@ -258,7 +259,7 @@ Pino.prototype.child = function child (bindings) {
   return new Pino(
     bindings.level || this.level,
     this.stream,
-    this.serializers,
+    extend(this.serializers, bindings.serializers),
```

Comment:
I think we should only allocate a new object if `bindings.serializers` is present.